const auth = (req, res, next) => {
    try {
        
    } catch (error) {
        throw error
        next()
    }
}

module.exports = auth